man/systemd.network.xml

```diff
@@ -637,6 +637,13 @@
         </para></listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>IPv6PrefixDelegation</varname></term>
```

Comment:
we try to write all config file options with a trailing "=" to indicate that they are config file options. Please follow this here, too, i.e. change IPv6PrefixDelegation → IPv6PrefixDelegation=